app/src/main/java/protect/card_locker/importexport/StocardImporter.java

```diff
@@ -233,22 +274,39 @@ public void importData(Context context, SQLiteDatabase database, File inputFile,
                 headerColor = Utils.getHeaderColorFromImage(cardIcon, headerColor);
             }
 
-            long loyaltyCardInternalId = DBHelper.insertLoyaltyCard(database, store, note, null, null, BigDecimal.valueOf(0), null, cardId, null, barcodeType, headerColor, 0, null,0);
+            LoyaltyCard card = new LoyaltyCard(tempID, store, note, null, null, BigDecimal.valueOf(0), null, cardId, null, barcodeType, headerColor, 0, Utils.getUnixTime(), DBHelper.DEFAULT_ZOOM_LEVEL, 0);
```

Comment:
Any reason why you changed the lastUsed from `0` to the current timestamp? The default DB value is `0`.